definitions/ext-service_level/summary_metrics.yml

```diff
@@ -12,10 +12,10 @@ sloPeriod:
 
 sliCompliance:
   goldenMetric: sliCompliance
-  title: SLI compliance
-  unit: PERCENTAGE
+  title: SLI compliance (%)
+  unit: STRING
```

Comment:
Metrics should not be strings. only tags can be strings. what is the reason for this change?

Reply:
we would like the rounding to not be done to 2 digits as we allow SLO to have up to 4 decimal digits. We thought that by using a string the rounding would not take effect

Reply:
I see, that seems to be a presentational issue that should be dealt with in the UI. We enforce metrics not to be a string to precisely avoid this issue - UIs should render them consistently based on the unit they have associated with.